src/plugins/collision/collider/mod.rs

```diff
@@ -420,7 +422,13 @@ impl From<Transform> for ColliderTransform {
 /// fn setup(mut commands: Commands) {
 ///     // Spawn a static body with a sensor collider.
 ///     // Other bodies will pass through, but it will still send collision events.
-///     commands.spawn((RigidBody::Static, Collider::ball(0.5), Sensor));
+///     let collider = Collider::ball(0.5);
+///     commands.spawn((
+///         RigidBody::Static,
+///         MassPropertiesBundle::new_computed(&collider, 1.0),
```

Comment:
Why does this sensor have a `MassPropertiesBundle`?

Reply:
Oops, somehow I didn't notice that it's static and just thought "oh no, this would be a rigid body with zero mass!" which would only be problematic if this was dynamic 😅 Fixing